buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -27,41 +14,37 @@ class Sprite:
     image: Optional[Surface] = None
     rect: Optional[Rect] = None
     def __init__(self, *groups: AbstractGroup) -> None: ...
-    def update(self, *args, **kwargs) -> None: ...
+    def update(self, *args: str, **kwargs: int) -> None: ...  # TODO: verify
```

Comment:
There's a "TODO: verify" here. 

Think it was ok as it was before. I don't think args is of type str, and kwargs is of type int?

Reply:
found issue 5876 in mypy where BDFL ok'ed it. Will revert